24life/blocks/article-hero-video/article-hero-video.css

```diff
@@ -48,6 +59,14 @@ div.article-hero-video-wrapper {
   letter-spacing: 1px;
 }
 
+.article-hero-video article-hero-video-title-newsletter {
+  font-size: 40px;
+  line-height: 1;
+  font-weight: 100;
+  margin-top: 0;
+  letter-spacing: 1px;
+}
+
```

Comment:
I would remove any custom styles for newsletter. My intention was to increase consistency. That means all hero will have the same visual look.

Reply:
sure